packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1751,6 +1751,7 @@
   "error-not-allowed": "Not allowed",
   "error-not-authorized": "Not authorized",
   "error-office-hours-are-closed": "The office hours are closed.",
+  "error-omnichannel-is-disabled": "Error! Omnichannel is disabled",
```

Comment:
is this going to be used somewhere?

Reply:
I realised this isn't getting used anywhere. So reverting it now